pkg/api/v1alpha1/nutanixdatacenterconfig_types.go

```diff
@@ -0,0 +1,140 @@
+// Important: Run "make generate" to regenerate code after modifying this file
+// json tags are required; new fields must have json tags for the fields to be serialized
+
+package v1alpha1
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NutanixDatacenterConfigSpec defines the desired state of NutanixDatacenterConfig
+type NutanixDatacenterConfigSpec struct {
+	// Endpoint is the Endpoint of Nutanix Prism Central
+	// +kubebuilder:validation:Required
+	Endpoint string `json:"endpoint"`
+
+	// Port is the Port of Nutanix Prism Central
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Minimum=9440
+	Port int `json:"port"`
+
+	// AdditionalTrustBundle is the trust bundle for Prism Centrals using non-publicly trusted certificates
+	// +kubebuilder:validation:Required
+	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
```

Comment:
We have `Required` and `omitempty` for this field. Does that work together?

Reply:
Good catch! Removing required. This field is for providing a cert bundle that you would like to trust in case of self-signed certs.